tests/unit/Framework/MockObject/GeneratorTest.php

```diff
@@ -240,4 +296,18 @@ public function testVariadicArgumentsArePassedToMockedMethod(): void
 
         $mock->foo(...$arguments);
     }
+
+    public function testGetClassMethodsWithNonExistingClass(): void
+    {
+        $this->expectException(\PHPUnit\Framework\MockObject\RuntimeException::class);
+
+        $this->generator->getClassMethods('Tux');
```

Comment:
Should we assert first (in the test) that the class really does not exist?

Reply:
good idea, fixed that